src/constants/api.tsx

```diff
@@ -2,3 +2,17 @@ export const API_KEY = 'api_key=157c2ade7d0b335008ae899a157d8967';
 export const API_URL = 'https://api.themoviedb.org/3/';
 export const API_IMG_URL = 'https://image.tmdb.org/t/p/';
 export const API_DATE_QUERY = 'primary_release_date.lte=';
+export const API_GENRE_ID = [
+    { comedy: 35 },
+    { drama: 18 },
+    { romance: 10749 },
+    { mystery: 9648 },
+    { thriller: 53 },
+    { animation: 16 },
+    { action: 28 },
+    { adventure: 12 },
+    { crime: 80 },
+    { fantasy: 14 },
+    { 'Science Fiction': 28 },
+    { Family: 10751 },
```

Comment:
I think we have Enum with categories? 
should be { [Genres.comedy]: 35, ....} as single object and not array